web/spacewalk-web.spec

```diff
@@ -32,6 +32,7 @@
 %endif
 %endif
 %{!?rhel: %global sbinpath /sbin}%{?rhel: %global sbinpath %{_sbindir}}
+%{!?nodejs_sitelib:%define nodejs_sitelib %{_prefix}/lib/node_modules}
```

Comment:
IIRC this PR correctly, you need to add the license from the old nodejs-sdk package to this package, right?

Also remember to remove the old nodejs-sdk packages from `rel-eng/packages`

Reply:
As similar to @agraul I just reviewed specs and changelogs, but not the rest of the changes, as I don't have enough knowledge to judge :-)